src/Container.php

```diff
@@ -57,26 +57,28 @@ public function start(): void
             '--http-enabled',
             '--https-enabled=false',
         ];
-        $content = [];
+
+        $container = (new GenericContainer("{$this->imageName}:{$this->imageTag}"))
+            ->withExposedPorts($this->internalPort)
+            ->withCommand($command);
 
         if ($this->signingKey instanceof SigningKey) {
+            // Create a temporary file with the signing key
+            $tempFile = tempnam(sys_get_temp_dir(), 'esdb_signing_key_');
+            if ($tempFile === false) {
+                throw new RuntimeException('Failed to create temporary file for signing key.');
+            }
+            file_put_contents($tempFile, $this->signingKey->privateKeyPem);
```

Comment:
This writes the key to the local file system.. It would be nice if that wasn't required, but I couldn't get it to work otherwise